src/context/index.js

```diff
@@ -3,8 +3,12 @@
 import AuthProvider from "./AuthProvider";
 import VideoProvider from "./VideoProvider";
 import LikeProvider from "./LikeProvider";
+import WatchLaterProvider from "./WatchLaterProvider";
+import HistoryProvider from "./HistoryProvider";
 export {useAuth} from "./AuthProvider"
 export {useVideos} from "./VideoProvider"
 export {useLikes} from "./LikeProvider"
+export {useWatchLater} from "./WatchLaterProvider"
```

Comment:
You can directly export directly by writing export {default as FileName} from FILE_PATH

Reply:
Hey didn't knew about this thanks for suggestions